crates/ethereum/node/tests/it/exex.rs

```diff
@@ -32,7 +32,7 @@ fn basic_exex() {
     let _builder = NodeBuilder::new(config)
         .with_database(db)
         .with_types::<EthereumNode>()
-        .with_components(EthereumNode::components())
+        .with_components::<_, EthereumAddOns>(EthereumNode::components())
```

Comment:
could we perhaps split this into two calls?
like `with_addons` I think this would still work because we can map the addons generic?

maybe not, then this is fine